database/user.go

```diff
@@ -635,254 +615,6 @@ func (db *DB) managedUserBySub(ctx context.Context, sub string) (*User, error) {
 	return &u, nil
 }
 
-// userStats reports statistical information about the user.
-func (db *DB) userStats(ctx context.Context, user User) (*UserStats, error) {
```

Comment:
Moving a ton of code around makes it hard to see what actually changed. For changes like this one I'd recommend first implementing the functional changes and then have a dedicated "just moving some code" MR.

Reply:
Fair. Will do next time.